flapp-webapp/vue.config.js

```diff
@@ -1,13 +1,21 @@
 module.exports = {
   chainWebpack: config => {
     config.module.rules.delete("eslint");
+    config.module.rule('ts')
+    .test(/\.ts$/)
+    .use('ts-loader')
+    .loader('ts-loader')
+    .options({
+      appendTsSuffixTo: [/\.vue$/]
+    })
   },
   css: {
     loaderOptions: {
       sass: {
         prependData: `@import "@/styles/_variables.scss";`,
         prependData: `@import "@/styles/_common.scss";`,
```

Comment:
It should not be necessary to prepend anything, especially not files that produce presentation rules as they will get copied everywhere. `styles/common` is meant to be imported in components that want to inherit the bootstrap utility functions and variables (no presentation rules), and it imports `variables`.  Styles that are used everywhere (`styles.scss`) should be imported in main.js.